test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -18,14 +18,23 @@
 import shutil
 import urllib.request
 import os
+import builtins
 
 
 @OnScenarioStart
 def hook(context):
     from configparser import ConfigParser
 
     cfg = ConfigParser()
-    cfg.read('../config.ini')
+
+    if os.environ.get('CI') in ['True', 'true']:
+        cfg.read('../drone/config.ini')
+    else:
+        cfg.read('../config.ini')
```

Comment:
don't do this here, if you want to use a specific ini file for drone, just copy it to `test/gui/config.ini` using the drone.star file